.travis.yml

```diff
@@ -41,8 +41,7 @@ install:
 # TF tests first, because more relevant (for me), then ordered by alphabet
 env:
   global:
-    # TODO: TF 1.14 needs multiple fixes...
-    - TF_PACKAGE=tensorflow==1.13.1
```

Comment:
I would move this change to an own commit, and it should be the last commit in this PR.

Reply:
No problem!